x-pack/plugins/security_solution/common/api/endpoint/actions/state/state.schema.yaml

```diff
@@ -15,5 +15,19 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '../../model/schema/common.schema.yaml#/components/schemas/SuccessResponse'
+                $ref: '#/components/schemas/ActionStateSuccessResponse'
+components:
+  schemas:
+    ActionStateSuccessResponse:
+        type:
+          object
+        properties:
```

Comment:
You might need to specify properties in `required` if these aren't optional.

Reply:
828bb2a756bba9c6c288033d073f33337e6725b4